msi/release.sh

```diff
@@ -185,7 +186,7 @@ cat <<EOF
     </Fragment>
 </Wix>
 EOF
-)>GitComponents.wxs
+) | sed -r -e "s/(<File .*Source=\")(.*(compat-)?(ba)?sh.exe|.*git.*\.exe)([^>]*) \/>/\1\2\" BindPath=\"[BindImagePaths]\5 \/>/" >GitComponents.wxs
```

Comment:
Ah, I thought `-r` was pretty portable by now. That's a good reason--I still don't like it 😉 OK, I'll not use `-r`.